test/data.jl

```diff
@@ -186,107 +184,91 @@ module TestData
-    # @test isequal(m2[:b2],
-    #               NullableArray(Nullable{String}["B", "B", "B", "C", "B",
-    #                                              Nullable(), Nullable(),
-    #                                              Nullable(), Nullable(), Nullable()]))
-
-    dt1 = DataTable(a = NullableArray([1, 2, 3]),
-                    b = NullableArray(["America", "Europe", "Africa"]))
-    dt2 = DataTable(a = NullableArray([1, 2, 4]),
-                    c = NullableArray(["New World", "Old World", "New World"]))
+    @test m2[:b2] == [null, :A, :A, null, :C, null, null, :B, null, :A]
+
+    dt1 = DataTable(a = [1, 2, 3],
+                    b = ["America", "Europe", "Africa"])
+    dt2 = DataTable(a = [1, 2, 4],
+                    c = ["New World", "Old World", "New World"])
 
     m1 = join(dt1, dt2, on = :a, kind = :inner)
-    @test isequal(m1[:a], NullableArray([1, 2]))
+    @test m1[:a] == [1, 2]
 
     m2 = join(dt1, dt2, on = :a, kind = :left)
-    @test isequal(m2[:a], NullableArray([1, 2, 3]))
+    @test m2[:a] == [1, 2, 3]
 
     m3 = join(dt1, dt2, on = :a, kind = :right)
-    @test isequal(m3[:a], NullableArray([1, 2, 4]))
+    @test m3[:a] == [1, 2, 4]
 
     m4 = join(dt1, dt2, on = :a, kind = :outer)
-    @test isequal(m4[:a], NullableArray([1, 2, 3, 4]))
+    @test m4[:a] == [1, 2, 3, 4]
 
     # test with nulls (issue #185)
     dt1 = DataTable()
-    dt1[:A] = NullableArray(["a", "b", "a", Nullable()])
-    dt1[:B] = NullableArray([1, 2, 1, 3])
+    dt1[:A] = ["a", "b", "a", null]
+    dt1[:B] = [1, 2, 1, 3]
 
     dt2 = DataTable()
-    dt2[:A] = NullableArray(["a", Nullable(), "c"])
-    dt2[:C] = NullableArray([1, 2, 4])
+    dt2[:A] = ["a", null, "c"]
+    dt2[:C] = [1, 2, 4]
 
     m1 = join(dt1, dt2, on = :A)
     @test size(m1) == (3,3)
-    @test isequal(m1[:A], NullableArray(["a","a", Nullable()]))
+    @test m1[:A] == ["a","a", null]
 
     m2 = join(dt1, dt2, on = :A, kind = :outer)
     @test size(m2) == (5,3)
-    @test isequal(m2[:A], NullableArray(["a", "b", "a", Nullable(), "c"]))
+    @test m2[:A] == ["a", "b", "a", null, "c"]
 
     srand(1)
     dt1 = DataTable(
-        a = NullableArray(rand([:x,:y], 10)),
-        b = NullableArray(rand([:A,:B], 10)),
-        v1 = NullableArray(randn(10))
+        a = rand([:x,:y], 10),
+        b = rand([:A,:B], 10),
+        v1 = randn(10)
     )
 
     dt2 = DataTable(
-        a = NullableArray([:x,:y][[1,2,1,1,2]]),
-        b = NullableArray([:A,:B,:C][[1,1,1,2,3]]),
-        v2 = NullableArray(randn(5))
+        a = Vector{Union{Symbol, Null}}([:x,:y][[1,2,1,1,2]]),
+        b = [:A,:B,:C][[1,1,1,2,3]],
+        v2 = randn(5)
     )
-    dt2[1,:a] = Nullable()
+    dt2[1,:a] = null
 
     # # TODO: Restore this functionality
     # m1 = join(dt1, dt2, on = [:a,:b])
-    # @test isequal(m1[:a], NullableArray(["x", "x", "y", "y", fill("x", 5)]))
+    # @test m1[:a] == Vector(["x", "x", "y", "y", fill("x", 5)]))
```

Comment:
Try to uncomment this test. If still fails, add more information to the TODO

Reply:
BTW, `Vector` isn't needed now.